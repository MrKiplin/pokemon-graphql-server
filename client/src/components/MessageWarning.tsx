import React from "react";
import { Message } from "semantic-ui-react";

interface MessageWarningProps {
  message: string;
}

const MessageWarning: React.FC<MessageWarningProps> = ({ message }) => (
  <Message warning>
    <Message.Header>Warning</Message.Header>
    <p>{message}</p>
  </Message>
);

export default MessageWarning;
