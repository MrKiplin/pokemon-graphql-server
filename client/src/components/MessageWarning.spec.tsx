import { mount } from "enzyme";
import MessageWarning from "./MessageWarning";

describe("MessageWarning", () => {
  it("Should match props and snapshot", () => {
    const component = mount(
      <MessageWarning {...{ message: "test-message" }} />
    );

    expect(component).toMatchSnapshot();
    expect(component.find("MessageWarning").prop("message")).toEqual(
      "test-message"
    );
  });
});
