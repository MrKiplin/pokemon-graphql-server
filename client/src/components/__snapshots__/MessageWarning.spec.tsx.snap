// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MessageWarning Should match props and snapshot 1`] = `ReactWrapper {}`;
